### PR TITLE
TW 375: Command Information

### PR DESCRIPTION
**Pull request by DJSanti**
## What was changed
More information about command functionality, along with code examples, have been added.

## Why?
CLI and Docs need more information.

## How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func commands(defaultCfg *sconfig.Config) []*cli.Command {
 	return append(
 		append(
 			serverCommands(defaultCfg),
-			common.WithFlags(clientCommands, common.SharedFlags)...,
+			common.WithFlags(clientCommands, common.FlagsForPaginationAndRendering)...,
```
Comment (feedmeapples) [inline app/app.go:90]: why changing this? 

Pagination And Rendering flags apply to some commands, not all client commands

File: cmd/docgen/main.go [hunk 2]
```diff
@@ -86,31 +87,34 @@ func main() {
...
 		} else if strings.HasPrefix(line, "**--") {
 			// split into term and definition
-			term, definition, found := strings.Cut(line, ":")
+			term, definition, _ := strings.Cut(line, ":")
 			term = strings.TrimSuffix(term, "=\"\"")
 			if strings.Contains(term, ",") {
 				termArray := strings.Split(line, ",")
 				optionFileName = termArray[0] + "**"
-				aliasName = "Alias: **" + strings.TrimSpace(termArray[1])
+				//aliasName := "Alias: **" + strings.TrimSpace(termArray[1])
```
Comment (feedmeapples) [inline cmd/docgen/main.go:95]: if this is not needed anymore let's clear it out

File: common/defs-cmds.go [hunk 3]
```diff
@@ -81,243 +81,328 @@ const (
...
+Use the command options below to change the information returned by this command.`
 
 const DescribeWorkflowUsageText = `The ` + "`" + `temporal workflow describe` + "`" + ` command shows information about a given [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
 This information can be used to locate Workflow Executions that weren't able to run successfully.
 
-Use the command options listed below to change the information returned by this command.
-Make sure to write the command in this format:
-` + "`" + `temporal workflow describe [command options]` + "`"
+` + "`" + `temporal workflow describe --workflow-id=meaningful-business-id` + "`" + `
```
Comment (feedmeapples) [inline common/defs-cmds.go:150]: does this look good when printed with --help from the binary? Making sure the extra lines do not mess up --help output

Comment (DJSanti) [reply common/defs-cmds.go:150]: yeah, it looked fine

**Comment by DJSanti** [comment]
Changing per comments. After this, consider deleting the tool and editing the docs when necessary.

File: common/defs-flags.go [hunk 4]
```diff
@@ -1,16 +1,16 @@
 package common
 const (
 	// Shared flag definitions
-	FlagEnvDefinition = "Name of the environment to read environmental variables from."
+	FlagEnvDefinition = "Environment to read environmental variables from."
```
Comment (bergundy) [inline common/defs-flags.go:4]: This sentence is confusing, let's get rid of the word environmental, it doesn't apply here.

**Pull request merged**